src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[11];
+        figures[0] = figureSupplier.getDefaultFigure();
+        Random random = new Random();
+        for (int i = 1; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure(random.nextInt(5));
```

Comment:
you should depend on one constant, don't hardcode its half, if the change will need to be made it will be in two places, instead of one - it's error-prone